.github/workflows/default_branch_verification.yml

```diff
@@ -17,5 +17,9 @@ jobs:
           java-version: '17'
           distribution: 'temurin'
           cache: gradle
+      - name: Read google-service.json
```

Comment:
It would be better to use a more descriptive commit message 😄 `Fast Fix Dev branch` is not saying anything expect that it is fixing something

Reply:
I hope it is better 
![image](https://github.com/intive/Picover/assets/57491794/dfb35ad2-411c-434f-8106-00e1c2367bee)
😄